Tomcat/Default Credential.md

```diff
@@ -23,10 +23,17 @@ for a vulnerability to have one which is beyond the scope of this database.
 ## Detection
 
 How does one detect the exploitation of this vulnerability, or detect its presence.
+username/password
```

Comment:
Please put "-" dash marks before each password so that it shows up as a bulleted list